pkg/performanceprofile/profilecreator/profilecreator.go

```diff
@@ -61,6 +61,21 @@ const (
 	allCores = -1
 )
 
+var (
+	noFilerLogicalProcessors = func(int, int) bool { return true }
```

Comment:
s/noFilerLogicalProcessors/noFil**t**erLogicalProcessors

Reply:
other suggestion : change var name to unfilteredLogicalProcessors / allLogicalProcessors / LogicalProcessors